### ISSUE TITLE
Converting @everyone and @role mentions.

### ISSUE THREAD

**Issue by zertap** [sm00th/bitlbee-discord#61]
If a mention applies to you, it should be converted to your nick (though still displaying the original intent).
For example. @everyone -> HL: your_nick @everyone ...

**Comment by zertap** [comment]
Of course, most likely the clients people use do allow adding additional highlights per-server or even per-channel, so adding them from the client wouldn't be a big hassle (actually done that already myself). But I decided to bring up this and a few other minor improvements that came to mind.

**Comment by zertap** [comment]
Actually, @role mentions seem are kind of hard to even figure out who they were meant for from irc side, since role highlights are displayed like `<@&280335162884227073>`
Converting these highlights would also allow to check if the user actually has the right to ping you this way.

**Comment by sm00th** [comment]
The whole role part of api is not implemented currently. Bitlbee-discord is completely oblivious of roles atm because there was no good reason apart from not joining the channels you are not supposed to. This is another reason, but I suspect that I'll implement `@everyone` way before the `@role` pings.

**Comment by zertap** [comment]
I feel like at this point this would be a really nice improvement.
I like how slack irc gateway adds "cc: your_nick" if the message should highlight you.
